.buildkite/premerge.steps.yaml

```diff
@@ -22,7 +22,13 @@ common: &common
 
 steps:
   - label: "build-GDK-:windows:"
-    command: "powershell ./ci/setup-and-build-gdk.ps1"
-    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede. 
+    command: "powershell ./ci/setup-and-build-gdk.ps1 -target_platform Win64"
+    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
+    artifact_paths:
+      - "UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
+
+  - label: "build-GDK-:linux:"
+    command: "powershell ./ci/setup-and-build-gdk.ps1 -target_platform Linux"
+    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
     artifact_paths:
-      - "UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
\ No newline at end of file
+      - "UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
```

Comment:
Super minor - looking at the Linux build ran in the last check here https://buildkite.com/improbable/unrealgdk-premerge/builds/991#d362a14f-31ef-4632-9ab8-27482caf9b02 it looks like the artifacts were uploaded under the name of UBT-UE4Editor-Win64-Development.txt - i assume this is due to the fact we're building on Windows but might be worth renaming the logs/changing the structure to make it clear which platform output log comes from where?

Reply:
Strangely, this seems resolved in the latest round of builds.